core/src/main/java/com/novoda/merlin/service/MerlinService.java

```diff
@@ -52,7 +52,7 @@ public void onConnectivityChanged(ConnectivityChangeEvent connectivityChangeEven
 
     public class LocalBinder extends Binder {
 
-        public ConnectivityChangesListener getConnectivityChangedListener() {
+        public ConnectivityChangesListener connectivityChangedListener() {
```

Comment:
Only accessible through the `ConnectivityChangeNotifier` clients should not have access to this, so we can remove the `get`.